util/Migrator/DbScripts/2020-11-16_00_SalesTax.sql

```diff
@@ -0,0 +1,153 @@
+IF OBJECT_ID('[dbo].[TaxRate]') IS NOT NULL
+BEGIN
+    DROP TABLE [dbo].[TaxRate]
+END
+GO
+
+CREATE TABLE [dbo].[TaxRate] (
+    [Id]                VARCHAR(40)         NOT NULL,
+    [Country]           VARCHAR(50)         NOT NULL,
+    [State]             VARCHAR(2)          NULL,
+    [PostalCode]        VARCHAR(10)         NOT NULL,
+    [Rate]              DECIMAL(5,2)        NOT NULL,
+    [Active]            BIT                 NOT NULL,
+    CONSTRAINT [PK_TaxRate] PRIMARY KEY CLUSTERED ([Id] ASC)
+);
+GO
```

Comment:
This `GO` statement goes away and instead, pull in the `CREATE UNIQUE INDEX` statement inside of the same "batch" (between GO's), all wrapped in that `IF` not exists condition.